web/react/utils/utils.jsx

```diff
@@ -124,8 +124,10 @@ module.exports.notifyMe = function(title, body, channel) {
 }
 
 module.exports.ding = function() {
-  var audio = new Audio('/static/images/ding.mp3');
-  audio.play();
+    if (!navigator || navigator.userAgent.toLowerCase().indexOf("firefox") === -1) {
```

Comment:
If you need to check that navigator.userAgent exists above, it should probably be checked here as well.